br/pkg/version/version.go

```diff
@@ -233,6 +233,15 @@ func CheckVersionForBR(s *metapb.Store, tikvVersion *semver.Version) error {
 			s.Address, tikvVersion)
 	}
 
+	// 8.2 br(store based backup) does not support tikv <= 8.1
+	// due to the performance issue https://github.com/tikv/tikv/issues/17168
+	// TODO: we can remove this check if the performance issue is fixed and cherry-pick
+	if (BRVersion.Major > 8 || (BRVersion.Major == 8 && BRVersion.Minor >= 2)) &&
```

Comment:
This check is always true?

Reply:
If this PR wasn't cherry-picked, it looks always true. But I guess this won't be really harmful.